cmd/config.go

```diff
@@ -96,14 +100,13 @@ func patchConfig(cliCtx *cli.Context, req map[string]*string) error {
 		return cli.NewExitError("Config can only be set on custom resources", -1)
 	}
 
-	spin := prompts.NewSpinner("Setting resource config")
-	spin.Start()
-	defer spin.Stop()
+	prompts.SpinStart("Setting resource config")
```

Comment:
wanna change this to "Modifying resource config" or similar? this func both sets and unsets

Reply:
I used `Update` on success, so will use `Updating resource config`